compose/production/nginx/default.conf

```diff
@@ -1,5 +1,5 @@
 server {
-   listen                      80; # listen on http
+   listen                      18080; # listen on http
```

Comment:
standard port when you can't 80 is 8080

Reply:
I know, but I don't see why it matters. :)